bin/docker/Dockerfile_ubuntu

```diff
@@ -0,0 +1,28 @@
+FROM eclipse-temurin:11-jre-jammy as runner
+
+ENV FC_LANG=en-US LC_CTYPE=en_US.UTF-8
+
+# Dependencies
+RUN apt-get update && \
+  apt-get upgrade -y && \
+  apt-get install -y ca-certificates ca-certificates-java && \
+  apt-get clean && \
+  curl https://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem -o /usr/local/share/ca-certificates/rds-combined-ca-bundle.pem && \
+  curl https://cacerts.digicert.com/DigiCertGlobalRootG2.crt.pem -o /usr/local/share/ca-certificates/DigiCertGlobalRootG2.crt.pem && \
+  update-ca-certificates && \
+  mkdir -p /plugins && chmod a+rwx /plugins && \
+  keytool -list -cacerts
+
+# add Metabase jar & add our run script to the image
+COPY ./metabase.jar ./run_metabase.sh /app/
+
+# expose our default runtime port
+EXPOSE 3000
+
+# if you have an H2 database that you want to initialize the new Metabase
```

Comment:
I would remove these comments from the Dockerfile, I prefer to have comments in the docs

Reply:
We have that comment in our current dockerfile for Alpine. I can remove it, but that is kind of an out of scope, I tried to save it as close to the current image as possible.